src/packages/ratmama/ratmama_parser.py

```diff
@@ -122,9 +124,12 @@
 
         ctx.bot.board.append(rescue, overwrite=False)
         index = ctx.bot.board.find_by_name(client=client_name).board_index
+        landmark = await Galaxy.find_nearest_landmark((await Galaxy.find_system_by_name(system_name)))
+        distance_str = f"{landmark[1]} LY from {landmark[0].name}" if landmark[1] != 0 else ""
+
         await ctx.reply(f"RATSIGNAL - CMDR {client_name} - "
-                        f"Reported System: {system_name} (distance to be implemented) - "
-                        f"Platform: {platform_name} - "
+                        f"Reported System: {system_name} ({distance_str})"
```

Comment:
[Error] If `find_nearest_landmark` does not return a distance greater than `0`, this string will interpolate to, for instance, `"Reported System: Sol ()"`